src/main/java/org/jitsi/videobridge/RtpChannel.java

```diff
@@ -900,10 +908,9 @@ void initialize(RTPLevelRelayType rtpLevelRelayType)
 
             // The transport manager could be already connected, in which case
             // (since we just created the stream), any previous calls to
-            // transportConnected() have failed started the stream. So trigger
+            // transportConnected() have failed to start the stream. So trigger
             // one now, to make sure that the stream is started.
-            TransportManager transportManager = getTransportManager();
-            if (transportManager != null && transportManager.isConnected())
+            if (transportManager.isConnected())
```

Comment:
Because Channel#transportManager, while not final, is always initialized to non-null in Channel#initialize()